src/components/Skills.tsx

```diff
@@ -1,33 +1,28 @@
-import SingleSkillTypeTable from "./SingleSkillTypeTable"
-import "../style/skills.css"
-
+import SingleSkillTypeTable from './SingleSkillTypeTable';

```

Comment:
Is it standard practice to use single quotes for the imports but double quotes for everything in the HTML bits or is that just George flavour?